server/db/index.js

```diff
@@ -4,6 +4,9 @@ module.exports = db;
 
 var User = require('./models/user');
 var Plant = require('./models/plant');
+var Plot = require('./models/plot');
 
-User.belongsToMany(Plant, {through: 'Plot'});
-Plant.belongsToMany(User, {through: 'Plot'});
+Plot.belongsToMany(Plant, {through: 'PlotPlants'});
```

Comment:
prefer underscore case for table names (e.g. `plot_plants`). Mixed case works in Sequelize because Sequelize double-quotes everything, but it's slightly annoying e.g. in psql or Postico etc.